admin/partials/settings.php

```diff
@@ -26,7 +26,7 @@
 				<tr>
 					<th scope="row">
 						<label for="gdpr_deletion_needs_review"><?php esc_html_e( 'User deletion', 'gdpr' ); ?>:</label>
-						<span class="screen-reader-text"><?php esc_attr_e( 'Useful if you need to remove the user from third-party services.', 'gdpr' ); ?></span>
+						<span class="screen-reader-text"><?php esc_html( 'Useful if you need to remove the user from third-party services.', 'gdpr' ); ?></span>
```

Comment:
Not sure this would apply translations? `esc_html` only takes 1 argument https://developer.wordpress.org/reference/functions/esc_html/. Think it's `esc_html_e` you're after

Reply:
Thanks, will swap them out. (Not many of the lines are in our translation file unfortunately)